### PR TITLE
Remove "boring" from the generator adjectives

### PR DESCRIPTION
**Pull request by stuartpb**
Beyond Wozniak, why would _anybody_ on this list deserve to be randomly considered "boring"?

### CODE REVIEW

**Comment by jfrazelle** [comment]
Can you please sign your commits following these rules:

https://github.com/docker/docker/blob/master/CONTRIBUTING.md#sign-your-work

The easiest way to do this is to amend the last commit:

`git commit --amend -s --no-edit && git push -f`

**Comment by crosbymichael** [comment]
If you take one out you have to put another back in, it's the rules ;)

File: pkg/namesgenerator/names-generator.go [hunk 1]
```diff
@@ -306,9 +305,6 @@ func GetRandomName(retry int) string {
 
 begin:
 	name := fmt.Sprintf("%s_%s", left[rand.Intn(len(left))], right[rand.Intn(len(right))])
-	if name == "boring_wozniak" /* Steve Wozniak is not boring */ {
```
Comment (jfrazelle) [inline pkg/namesgenerator/names-generator.go]: but this gives docker some charm no?

**Comment by tianon** [comment]
How is `boring` any more or less applicable to this list than `evil`, `high`, `goofy`, `tender`, etc?  Everybody's boring sometimes (except Woz, but that's the joke).

**Comment by jfrazelle** [comment]
serious

**Comment by vieux** [comment]
NOT LGTM

**Comment by tiborvass** [comment]
Sorry @stuartpb I agree with others that `boring` is not a special case. A better proposal could be to remove all the negative adjectives, but we would have to add as many positive adjectives.

**Comment by icecrime** [comment]
And maybe `Einstein` was never `drunk`, but I like my silly output :-) Not LGTM!

**Comment by jfrazelle** [comment]
Going to close but you can open a new one w someone awesome or a new adjective!
Also I bet Linus torvalds is pretty boring at times

**Comment by tianon** [comment]
Imagine him actually doing the Zombie Shuffle:

**Pull request closed without merging**